src/java/org/apache/cassandra/config/Config.java

```diff
@@ -380,9 +380,23 @@
     public RepairCommandPoolFullStrategy repair_command_pool_full_strategy = RepairCommandPoolFullStrategy.queue;
     public int repair_command_pool_size = concurrent_validations;
 
-    // parameters to adjust how much to delay startup until a certain amount of the cluster is connect to and marked alive
-    public int block_for_peers_percentage = 70;
+    /**
+     * When a node first starts up it intially considers all other peers as DOWN, and then as the initial gossip
+     * broadcast messages comes back nodes transition to UP. These options configure how long we wait for peers to
+     * connect before we make this node available as a coordinator. Furthermore, if this feature is enabled
+     * (timeout >= 0) Cassandra initiates the non gossip channel internode connections on startup as well and waits
```

Comment:
This part of the description doesn't match the description for block_for_peers_timeot_in_secs. It sounds like now it differentiates between disabled, and prime the connections.

Reply:
Ok, I reworded it. I think that having an escape hatch makes sense, what do you think?